dapp_runner/_util.py

```diff
@@ -53,3 +55,19 @@ def json_encoder(obj: Any):
         return obj.name
 
     return obj
+
+
+async def cancel_and_await_tasks(*tasks: Task) -> None:
+    """Cancel and await for cleanup of provided tasks."""
```

Comment:
nitpicking: grammatically, you "awaits something", not "await _for_ something" ...